VK_App/VK_App/ViewControllers/SecondBarVC/NewsHeaderCell.swift

```diff
@@ -0,0 +1,37 @@
+// NewsHeaderCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+///  Ячейка хедера
+final class NewsHeaderCell: UITableViewCell, NewsConfigurable {
+    
+    // MARK: Constants
+    private enum Constants {
+        static let dateStyle = "MM-dd-yyyy HH:mm"
+    }
+    // MARK: Private IBOtlets
+
+    @IBOutlet private var userImageView: UIImageView!
+    @IBOutlet private var userNameLabel: UILabel!
+    @IBOutlet private var datePostLabel: UILabel!
+
+    // MARK: Public Methodds
+
+    func configure(news: NewsFeed, network: NetworkService) {
+        datePostLabel.text = convertDate(dateValue: news.date)
+        userNameLabel.text = news.authorName
+        guard let avatarPost = news.avatarPath else { return }
+        userImageView.loadDatas(url: avatarPost, networkService: network)
```

Comment:
```suggestion
        userImageView.loadDatas(url: avatarPost)
```
NetworkService должен быть внутри сервиса по получению фото спроси у ребят в группе